### ISSUE TITLE
FavorUnlessOverNegatedIf triggered when using elsifs

### ISSUE THREAD

**Issue by jurriaan** [rubocop/rubocop#427]
Here's an example:

``` ruby
test = 'String'

a = if test.is_a?(String)
  3
elsif test.is_a?(Array)
  2
elsif !test.nil?
  1
end

p a
```

which gives the following output:

```
Offences:

test.rb:7:1: C: FavorUnlessOverNegatedIf: Favor unless (or control flow or) over if for negative conditions.
elsif !test.nil?
^^^
test.rb:9:1: W: EndAlignment: end at 9, 0 is not aligned with if at 3, 4
end
^^^

1 file inspected, 2 offences detected
```

How to fix this? I think it `FavorUnlessOverNegatedIf` shouldn't trigger in this case.
I also think the `EndAlignment` shouldn't trigger either.
